xmipp

```diff
@@ -850,7 +850,7 @@ def downloadDeepLearningModels(dest, dedicatedMode=False):
         return True
     url = "http://scipion.cnb.csic.es/downloads/scipion/software/em"
 
-    if not os.path.exists('bin/xmipp_sync_data'):
```

Comment:
I need further explanation on this. If that directory was wrong before, why isn't it causing problems for everyone?

Reply:
Nobody use it?, on devel it works for you?

Reply:
```
./xmipp get_models
 Xmipp has not been installed. Please, first install it 
```
after the change:
```
./xmipp get_models
Downloading Deep Learning models (in backgound)
Retrieving MANIFEST file
Fetching files...
 10%...
 20%...
...
```
Looks like it solves the issue.